cmd/svcat/output/binding.go

```diff
@@ -40,8 +42,17 @@ func getBindingStatusFull(status v1beta1.ServiceBindingStatus) string {
 	return formatStatusFull(string(lastCond.Type), lastCond.Status, lastCond.Reason, lastCond.Message, lastCond.LastTransitionTime)
 }
 
-// WriteBindingList prints a list of bindings.
-func WriteBindingList(w io.Writer, bindings ...v1beta1.ServiceBinding) {
+func writeBindingListJSON(w io.Writer, bindingList *v1beta1.ServiceBindingList) {
```

Comment:
What do you think about following the pattern set already for outputting yaml and then adding a similar method for json? Not sure if you tried that first and it didn't work well, but I'm pointing it out in case you didn't see it.

https://github.com/kubernetes-incubator/service-catalog/blob/b1da783cee7f0fbd70e4354fff61615ce3dbb546/cmd/svcat/output/plan.go#L130-L133

https://github.com/kubernetes-incubator/service-catalog/blob/2e1ca7f8cf2b5d45b0542602f0d12cb18bb05f98/cmd/svcat/output/yaml.go#L31-L45